otodom/task_1/Valentyn/src/Scraper.py

```diff
@@ -0,0 +1,82 @@
+import requests
+from bs4 import BeautifulSoup, Tag
+from HouseItem import HouseItem
+
+
+class Scraper:
+    TARGET_URL = "https://www.otodom.pl/pl/wyniki/sprzedaz/mieszkanie/mazowieckie/warszawa/warszawa?limit=72"
+    headers = {
+        "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.3"
+    }
+
+    def __init__(self):
+        self.itemLinksVisited = []
+
+    def startScraping(self, limit: int = 200) -> [HouseItem]:
+        response = requests.get(self.TARGET_URL, headers=self.headers)
+        if response.status_code // 100 != 2:
```

Comment:
why would you divide it by 100?

Reply:
Here I check the status code from the http request: if it 2xx (success) - than there were no  errors.
If it isn't,  we could react somehow to it (e.g. not parse the page which returned 404).

Reply:
the server shouldn't response with other code than 200 as long as you are not doinng any post/login requests. but fine